observations/obs_converters/MPD/work/mkmf_preprocess

```diff
@@ -0,0 +1,18 @@
+#!/bin/csh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
+
+../../../../build_templates/mkmf -p preprocess -t ../../../../build_templates/mkmf.template \
+ -a "../../../.." path_names_preprocess
+
+exit $status
+
+# <next few lines under version control, do not edit>
+# $URL$
+# $Revision$
+# $Date$
+
```

Comment:
At some point in my life, I am going to remove all these, so we don't need to add them to new files.
Please remove (and the DART $Id line at the top).  Thanks.

Reply:
i have removed all these